src/com/twu/biblioteca/MainMenu.java

```diff
@@ -45,26 +46,26 @@ public void showNoBooksMessage() {
 
 
     public void showOptions() {
-        for (int i = 0; i < options.size(); i++) {
-            System.out.println(options.get(i));
+        System.out.print(selectOption + "\n");
+        for (String option : options) {
+            System.out.println(option);
         }
         System.out.print(">");
     }
 
     public void manageOptions() {
-        int counter = 0;
         Scanner optionsScanner = new Scanner(System.in);
-
-        while (counter == 0) {
+            while (true) {
             if(optionsScanner.hasNext()) {
             String userInput = optionsScanner.next();
-            if (userInput.equals("1")) {
+            String cleanUserInput = userInput.trim();
+            if (cleanUserInput.equals("1")) {
                 showBookList();
                 showOptions();
-            } else if (userInput.equals("2")) {
+            } else if (cleanUserInput.equals("2")) {
                 showBookList();
                 manageBookList();
-            } else if (userInput.equals("3")) {
+            } else if (cleanUserInput.equals("3")) {
                 showGoodbyeMessage();
                 System.exit(0);
             } else {
```

Comment:
A question for Future You: Is there a point at which these ifs would be better implemented as a `switch`?